include/nil/actor/math/polynomial/polynomial_dfs_view.hpp

```diff
@@ -508,8 +508,7 @@ namespace nil {
                     return *this;
                 }
 
-                template<typename ContainerType>
-                void from_coefficients(const ContainerType &tmp) {
+                void from_coefficients(const container_type &tmp) {
```

Comment:
I think in some future we need creating polynomial from any [containers type](https://cplusplus.com/reference/stl/). That's why here was template not equal to internal container type.

Reply:
Undid this change.